source/unified-test-format/tests/invalid/expectedEventsForClient-eventType-enum.yml

```diff
@@ -0,0 +1,16 @@
+description: expectedEventsForClient-eventType-enum
+
+schemaVersion: '1.3'
+
+createEntities:
+  - client:
+      id: &client0 client0
+
+tests:
+  - description: invalid eventType value
+    operations: []
+    expectEvents:
+      - client: *client0
+        eventType: foo
+        events:
+          - commandStartedEvent: {}
```

Comment:
I suggest using an empty array for `events` here. That guarantees that the only schema validation error is the `eventType` value and not a `oneOf` requirement that enforces consistency between `eventType` and `events`.